.github/workflows/common.yml

```diff
@@ -122,7 +122,7 @@ jobs:
     with:
       r-version: "${{ needs.get_r_version.outputs.r-version }}"
   # vbump:
-  #   name: Version Bump 🤜🤛
+  #   name: Version Bump
```

Comment:
Why is this file updated? It shouldn't be part of the package. Thus it shouldn't cause any issues in the CRAN checks.

Reply:
These are the only other non-ascii characters, i see your point but it's also simpler to just remove all of them!!